app/settings.py

```diff
@@ -17,6 +18,14 @@ def get_key(key_name):
     vhost=os.getenv('SEFT_RABBITMQ_DEFAULT_VHOST', '%2f')
 )
 
+RABBIT_HEALTHCHECK_URL = "http://{hostname}:{port}/api/healthchecks/node".format(
+    hostname=os.getenv('SEFT_RABBITMQ_HOST', 'localhost'),
+    port=os.getenv('SEFT_RABBITMQ_PORT', 5672)
```

Comment:
Is this correct, i thought the HTTP port was 15672?

If so, it'll need a separate configuration value.